webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/uploading/UploadService.scala

```diff
@@ -31,8 +31,8 @@ import scala.concurrent.ExecutionContext
 case class ReserveUploadInformation(
     uploadId: String, // upload id that was also used in chunk upload (this time without file paths)
     name: String, // dataset name
-    directoryName: String, // dataset directory name
-    newDatasetId: String,
+    directoryName: Option[String], // dataset directory name
+    newDatasetId: Option[String],
     organization: String,
     totalFileCount: Long,
     filePaths: Option[List[String]],
```

Comment:
And the other occurence is here, where I accidentally made the new upload info a requirement which would have crashed uploads tried with wklibs